scss/utilities/_json-api.scss

```diff
@@ -0,0 +1,188 @@
+// JSO Export Utilities
+// --------------------
+
+
+/// ## Exporting Styles to JSON
+///
+/// While many UX patterns will be rendered as HTML components,
+/// there are several abstract style patterns –
+/// like color-palettes and font-specimens –
+/// that never appear as comonents in the application.
+///
+/// Herman provides several `@preview` annotations
+/// to help visualize these abstract patterns –
+/// but that requires passing the data from your styles
+/// to Herman's templates.
+/// We do that using Sass maps
+/// (key:value object variables)
+/// and the [`sass-json-export`][json] toolkit
+/// to generate JSON out of Sass variables.
+///
+/// [json]: https://github.com/oddbird/sass-json-export
+///
+/// @group api_json-export
+/// @link https://github.com/oddbird/sass-json-export Sass-JSON-Export
+///
+/// @example scss
+///   @import 'sass-json-export/stylesheets/sass-json-export';
+///
+///   @include json-encode($herman, comment);
```

Comment:
Is it confusing to include this as the first example, when we don't actually want users to use `sass-json-export` directly like this?